API.md

```diff
@@ -377,47 +401,79 @@ Example Response:
+| `/set.action` | `{JSON}` | 200 `Location: http://devstore.rerum.io/v1/id/22222` `{JSON}`
 
 - **`{JSON}`**—The requested new state for the object MUST contain an `@id`
 - **Response: `{JSON}`**—Containing various bits of information about the PATCH update. (see PUT Update for example)
 
 A single object is updated by adding all properties in the JSON
-payload. If a property already exists, a warning is returned to the user. 
+payload. If a property already exists, a warning is returned to the user. This
+is a specialized PATCH update with the same request, response, and history behavior.
+
 
 ### Remove Properties
 
 | Patterns | Payloads | Responses
 | ---     | ---     | ---
-| `/unset.action` | `{JSON}` | 202: `header.Location` New state `{JSON}`
+| `/unset.action` | `{JSON}` | 202 `Location: http://devstore.rerum.io/v1/id/22222` `{JSON}`
 
 - **`{JSON}`**—The requested new state for the object.  Must contain an `@id`.
 - **`{JSON}`**—Containing various bits of information about the PATCH update. (see PUT Update for example)
 
 A single object is updated by dropping all properties
 in the JSON payload list like `{key:null}`. Keys must match
-to be dropped otherwise a warning is returned to the user.
+to be dropped otherwise a warning is returned to the user.  This
+is a specialized PATCH update with the same request, response, and history behavior.
+
 
 ### RERUM released
 
 | Patterns | Payloads | Responses
 | ---     | ---     | ---
-| `/release.action` | `String @id` or `{JSON}` | 200: `header.Location` New state `{JSON}`
+| `/release.action` | `String @id` or `{JSON}` | 200 `Location: http://devstore.rerum.io/v1/id/11111` `{JSON}`
 
 - **`String @id`**—The `@id` of the version to be released.
 - **`{JSON}`**—The object.  Must contain `@id`.
 
-RERUM allows for the `generatedBy` of a version of an object to assign a `released` state. Objects in released states are locked such that further changes are refused. Calling any update or delete action on a released object will result in an error response. The release action will perform an update to the `__rerum.isReleased` of the version identified and update `__rerum.releases` properties throughout the object's history. Any version of an object with an `oa:Motivation` containing `rr:releasing` will be released as soon as it is saved.
+RERUM allows for the `generator` of a version of an object to assign a `released` state. 
+Objects in released states are locked such that further changes are refused. 
+Calling any update or delete action on a released object will result in an error response. 
+The release action will alter the `__rerum.isReleased` of the version identified 
+and alter `__rerum.releases` properties throughout the object's history without making a 
+new history state for the resulting object (the @id does not change). Any version of an object 
+with an `oa:Motivation` containing `rr:releasing` will be released as soon as it is saved.

```

Comment:
Do we actually do this?

Reply:
I believe so, at least remembering testing it I believe that's what it did.  We don't have an app that uses this behavior.  I did not test as I fixed this paragraph, but I did study the function a couple times and believe we actually do this.